From 2f0d6a6639ce6fc9f58e850a78416ed9ac5fba52 Mon Sep 17 00:00:00 2001
From: Tobias Fischer <info@tobiasfischer.info>
Date: Fri, 3 Jul 2026 09:20:00 +1000
Subject: [PATCH 3/3] do not vendor SDL2

---
 pyproject.toml | 2 --
 1 file changed, 2 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index e98d8d1..ce7efa5 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -70,7 +70,6 @@ norecursedirs = ["vcpkg", "build"]
 [tool.scikit-build]
 cmake.args = [
     "-DSDL_SUPPORT=ON",
-    "-DSDL_DYNLOAD=ON",
     "-DBUILD_CPP_LIB=OFF",
     "-DBUILD_PYTHON_LIB=ON",
     "-DBUILD_VECTOR_LIB=ON"
@@ -86,7 +85,6 @@ metadata.version.regex = "(?P<value>.+)"
 if.platform-system = "^(win32|linux)"
 cmake.args = [
     "-DSDL_SUPPORT=ON",
-    "-DSDL_DYNLOAD=ON",
     "-DBUILD_CPP_LIB=OFF",
     "-DBUILD_PYTHON_LIB=ON",
     "-DBUILD_VECTOR_LIB=ON",
