From f15d7662fe1c25e9a023ed24b807ef733d5733e0 Mon Sep 17 00:00:00 2001
From: Tobias Fischer <info@tobiasfischer.info>
Date: Wed, 7 Dec 2022 12:15:48 +1000
Subject: [PATCH 4/5] fix python executable

---
 setup.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/setup.py b/setup.py
index a7881ea..cb85844 100644
--- a/setup.py
+++ b/setup.py
@@ -38,6 +38,7 @@ class CMakeBuild(build_ext):
 
         cmake_args = [
             f"-DCMAKE_BUILD_TYPE={config}",
+            f"-DPython_EXECUTABLE={sys.executable}",
             f"-DPython3_EXECUTABLE={sys.executable}",
             f"-DCMAKE_LIBRARY_OUTPUT_DIRECTORY={extdir}",
             "-DSDL_SUPPORT=ON",
