From 1d65eb69fcb28c1f0083596ba7bbc36f0f93bc4c Mon Sep 17 00:00:00 2001
From: Tobias Fischer <info@tobiasfischer.info>
Date: Thu, 26 Sep 2024 14:19:46 +1000
Subject: [PATCH] Allow numpy 2.0

---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 75cff00ca..fd097be61 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -37,7 +37,7 @@ classifiers = [
     "Topic :: Scientific/Engineering :: Artificial Intelligence",
 ]
 dependencies = [
-  "numpy<2.0",
+  "numpy",
   "importlib-metadata>=4.10.0; python_version < '3.10'",
   "importlib-resources; python_version < '3.9'",
   "typing-extensions; python_version < '3.11'"
